src/components/one-time-donation/steps/SecondStep.tsx

```diff
@@ -15,8 +17,13 @@ export default function SecondStep() {
   const { data: session } = useSession()
 
   const [value, setValue] = useState('1')
-
+  const formik = useFormikContext<OneTimeDonation>()
   const handleChange = (event: React.SyntheticEvent, newValue: string) => {
+    if (Number(newValue) === 3) {
```

Comment:
@dimitur2204 It's not really clear to me what the constant `3` stands for. 

If we're using that as a number of step let's name it in some kind of enum or similar so it brings more context.

Similar to:

```tsx
if (Number(step) === Steps.Anonymous) {
  // ...
}
```

Reply:
As I see we can also use names instead of numbers for the tabs.

Ex. `logged`, `newProfile`, `anonymous` :thought_balloon: